docroot/themes/humsci/humsci_colorful/humsci_colorful.theme

```diff
@@ -23,6 +23,13 @@ function humsci_colorful_preprocess_html(&$vars) {
     // Set html_attributes for html DOM element
     $vars['html_attributes']->addClass('hb-dark-inversion');
   }
+
+  // Mega Menu hides standard menu on desktop setting.
+  $use_megamenu = theme_get_setting('megamenu_toggle');
+  if ($use_megamenu) {
+    // Load base megamenu CSS when megamenu is enabled
+    $vars['#attached']['library'][] =  'humsci_colorful/megamenu';
```

Comment:
This should be added at the block layer using a `preprocess_block__we_megamenu` I believe. Then you don't need to have the condition if the theme has it turned on

Reply:
alternatively, you could alter the library using a `hook_library_info_alter` and add the dependency